iocage_lib/ioc_fstab.py

```diff
@@ -378,3 +400,29 @@ def fstab_list(self):
         table.add_rows(flat_fstab)
 
         return table.draw()
+
+    def __fstab_encode__(self, _string):
+        """
+        Will parse any non-acceptable fstab characters and encode them
+
+        Example: ' ' -> \040
+        """
+        result = ctypes.create_string_buffer(1024)
```

Comment:
Are you sure it will always be enough?

Reply:
That's fstab's max length, so it should be

Reply:
`The size of dst must be four times the number of characters encoded from src (plus one for the NUL).` - from the manpage. It'll be much safer if we use this calculation instead of `1024`.

Reply:
@skarekrow Yes, but value passed to `__fstab_encode__` does not come from fstab and it will cause segmentation fault if it won't fit.